firmware/arduino/chessboard/uart.ino

```diff
@@ -1,84 +1,92 @@
 char incomingByte;
 char  buffer[5];
-int byteNum = -1; //-1 indicates that the start code hasn't been received
+int byteNum = -1; // -1 indicates that the start code hasn't been received
 
-//Wait for input
+// Wait for input
 void serialEvent2()
 {
-    //Loop through all available bytes
+    // Loop through all available bytes
     while (Serial2.available())
     {
-        //Get byte
+        // Get byte
         incomingByte = (char) Serial2.read();
 
-        //Reset buffer position
+        // Reset buffer position
         if (incomingByte == '~')
         {
             byteNum = 0;
         }
-        //Add byte to buffer
+        // Add byte to buffer
         else if (byteNum != -1)
         {
             buffer[byteNum++] = incomingByte;
         }
 
-        //Check if the buffer is full, process input
+        // Check if the buffer is full, process input
         if (byteNum == 5)
         {
-            //Reset buffer position
+            // Reset buffer position
             byteNum = -1;
 
-            //Process input
-            //Returns true for valid input and false for invalid input. Calls movement function
+            // Process input
+            // Returns true for valid input and false for invalid input. Calls movement function
             parseMessageFromPi(buffer);
         }
     }
 }
 
 bool parseMessageFromPi(char * message)
 {
-    //Move types 0 and 3 are valid for the same range of values
+    // Move types 0 and 3 are valid for the same range of values
     if (message[0] == 0 || message[0] == 3)
     {
-        //Check that the inputs are in a valid range
-        if (message[1] < 'a' || message[1] > 'h' || message[2] < '1' ||  message[2] > '8' || message[3] < 'a' || message[3] > 'h' || message[4] < '1' ||  message[4] > '8') 
+        // Check that the inputs are in a valid range
+        if (message[1] < 'a' || message[1] > 'h' 
+           || message[2] < '1' ||  message[2] > '8' 
+           || message[3] < 'a' || message[3] > 'h' 
+           || message[4] < '1' ||  message[4] > '8') 
         {
             return false;
         }
 
-        //Move type 0
+        // Move type 0
         if (message[0] == 0)
             moveStraight(message[1] - 'a', message[2] - '1', message[3] - 'a', message[4] - '1');
-        //Move type 3
+        // Move type 3
         else
             moveAlongEdges(message[1] - 'a', message[2] - '1', message[3] - 'a', message[4] - '1');
 
     }
     else if (message[0] == 1)
     {
-        //Check validity for the first 3 bytes
-        if (message[1] < 'a' || message[1] > 'h' || message[2] < '1' ||  message[2] > '8' || ((message[3] != 'b') && message[3] != 'w'))
+        // Check validity for the first 3 bytes
+        if (message[1] < 'a' || message[1] > 'h' 
+           || message[2] < '1' ||  message[2] > '8' 
+           || ((message[3] != 'b') && message[3] != 'w'))
```

Comment:
There are two extra sets of parenthesis here. They should be removed since the arguments are the same format as the rest of the comparisons and none of the other ones have these wack parenthesis.